### PR TITLE
docs(guide): improve url routing example readability

### PR DESCRIPTION
**Pull request by mthuret**
image

### CODE REVIEW

File: docgen/src/guides/advanced-topics.md [hunk 1]
```diff
@@ -101,18 +101,25 @@ import qs from 'qs';
 class App extends Component {
    constructor (props) {
            super(props);
-           this.state = {state: {...qs.parse(this.props.router.location.query)}}; //we initialize the state by parsing the url
+           //we initialize the state by parsing the url
```
Comment (vvo) [inline docgen/src/guides/advanced-topics.md]: missing space before we

File: docgen/src/guides/advanced-topics.md [hunk 2]
```diff
@@ -101,18 +101,25 @@ import qs from 'qs';
...
+           this.state = {state: {...qs.parse(this.props.router.location.query)}}; 
    }
     
     /*
-    push the new state to the react-router history. The threshold is there to specify how long we should wait between
-    state changes before pushing a new location instead of replacing the old one. This is a very basic implementation you
-    might want to perform advanced behaviors like removing empty values from the url or being able to keep others query params. 
+    push the new state to the react-router history. The threshold is there
```
Comment (vvo) [inline docgen/src/guides/advanced-topics.md]: Missing capital P

File: docgen/src/guides/advanced-topics.md [hunk 3]
```diff
@@ -101,18 +101,25 @@ import qs from 'qs';
 class App extends Component {
    constructor (props) {
            super(props);
```
Comment (vvo) [inline docgen/src/guides/advanced-topics.md]: Indent is double here

**Comment by mthuret** [comment]
Thanks for the typo @vvo 👍

**Pull request merged**